Makefile

```diff
@@ -10,11 +10,13 @@ install:
 	@mkdir -p "$(LIBDIR)"
 	@cp libs preload.lua .hilbishrc.lua "$(LIBDIR)" -r
 	@echo "Hilbish Installed"
+	@echo /usr/bin/hilbish >> /etc/shells
 
 uninstall:
 	@rm -vrf \
 			"$(BINDIR)/hilbish" \
 			"$(LIBDIR)"
+	@sed '/\/usr\/bin\/hilbish/d' /etc/shells
```

Comment:
wouldn't just an echo and redirect to append work
`echo a >> file`

Reply:
This is to remove it